test/test_force_align.py

```diff
@@ -52,6 +55,35 @@ def test_align_text(self):
         for w, xw in zip(words, xml_words):
             self.assertEqual(xw.attrib["id"], w["id"])
 
+    def test_align_switch_am(self):
```

Comment:
Thanks for increasing coverage. It always reassures me when stuff is better tested.